endpoints/cookie_sync_test.go

```diff
@@ -329,13 +385,24 @@ func TestCookieSyncHandle(t *testing.T) {
 			metrics:         &mockMetrics,
 			pbsAnalytics:    &mockAnalytics,
 			accountsFetcher: &fakeAccountFetcher,
+			time:            &fakeTime{time: time.Now()},
 		}
 		assert.NoError(t, endpoint.config.MarshalAccountDefaults())
 
 		endpoint.Handle(writer, request, nil)
 
 		assert.Equal(t, test.expectedStatusCode, writer.Code, test.description+":status_code")
 		assert.Equal(t, test.expectedBody, writer.Body.String(), test.description+":body")
+
+		gotCookie := writer.Header().Get("Set-Cookie")
+		if test.expectedCookieDeprecationHeader {
+			wantCookieTTL := endpoint.time.Now().Add(time.Second * time.Duration(86400)).UTC().Format(http.TimeFormat)
+			wantCookie := fmt.Sprintf("receive-cookie-deprecation=1; Path=/; Expires=%v; HttpOnly; Secure; SameSite=None; Partitioned;", wantCookieTTL)
+			assert.Equal(t, wantCookie, gotCookie, ":set_cookie_deprecation_header")
+		} else {
+			assert.Equal(t, gotCookie, "")
```

Comment:
Consider using `assert.Empty` here to be more declarative.

Reply:
done